test/graphql/mutations/delete_profile_mutation_test.rb

```diff
@@ -3,23 +3,29 @@
 require 'test_helper'
 
 class DeleteProfileMutationTest < ActiveSupport::TestCase
-  test 'delete a profile provided an ID' do
-    query = <<-GRAPHQL
-        mutation DeleteProfile($input: deleteProfileInput!) {
-            deleteProfile(input: $input) {
-                profile {
-                    id
-                }
-            }
-        }
-    GRAPHQL
+  QUERY = <<-GRAPHQL
+      mutation DeleteProfile($input: deleteProfileInput!) {
+          deleteProfile(input: $input) {
+              profile {
+                  id
+              }
+          }
+      }
+  GRAPHQL
 
+  setup do
     users(:test).update account: accounts(:test)
     profiles(:one).update(account: accounts(:test))
+    result = test_results(:one).update(
+      profile: profiles(:one), host: hosts(:one)
+    )
+    result.save
```

Comment:
Locally, some of these tests won't succeed, while on Jenkins this raises errors, with results appearently just being a trueclass....
```
ERROR["test_delete_a_profile_and_test_results_when_deleteAllTestResults_is_set", #<Minitest::Reporters::Suite:0x000055bec18afec8 @name="DeleteProfileMutationTest">, 4.455383786000311]

 test_delete_a_profile_and_test_results_when_deleteAllTestResults_is_set#DeleteProfileMutationTest (4.46s)

NoMethodError:         NoMethodError: undefined method `save' for true:TrueClass
...
```